angular-ngrx-scss/src/app/repository/components/repo-header/repo-heading/repo-heading.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="heading">
+  <span class="icon" size="24" appOcticon="repo"></span>
+  <div class="heading__container">
+    <a [routerLink]="ownerPath">{{ owner }}</a>
+    <span class="separator">/</span>
```

Comment:
Nitpick: no css is being set for the .separator class, so it's technically not needed.